Userland/Libraries/LibWeb/CSS/Parser/Parser.cpp

```diff
@@ -1934,8 +1934,14 @@ RefPtr<CSSRule> Parser::convert_to_rule(NonnullRefPtr<StyleRule> rule)
 
             auto media_query_tokens = TokenStream { rule->prelude() };
             auto media_query_list = parse_a_media_query_list(media_query_tokens);
+            if (!media_query_list.size()) {
+                return {};
+            }
```

Comment:
`if (media_query_list.is_empty())` would be clearer here. :^) Also, please remove the braces when it's a single line.